lib/problem.js

```diff
@@ -70,7 +70,10 @@ const problems = {
   },
   internal: {
     // no detail information, as this is only called when we don't know what happened.
-    unknown: problem(500.1, () => 'An unknown internal problem has occurred. Please try again later.')
+    unknown: problem(500.1, () => 'An unknown internal problem has occurred. Please try again later.'),
+
+    // { table: "the table missing a row", row: "a descriptor of the missing row" }
+    missingSystemRow: problem(500.2, ({ table }) => `Could not find an expected system ${table} record in your database.`)
```

Comment:
Does this problem not take a `row` argument? The comment above suggests it does, and I see that it's passed one from the CLI, but I don't see it actually using one.

Reply:
All parameters are passed through to the databag. Not all parameters are always used in the message generation.